### DISCUSSION TITLE
Update documentation regarding `queued` `workflow_job` hook

### DISCUSSION THREAD

**Discussion by matejdro** [community/community#127157] [Actions]
### Select Topic Area

Product Feedback

### Body

According to the https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=queued#workflow_job, `queued` is only triggered when `A job in a workflow run was created.`

However, according to the Github's support, `queued` is actually triggered for a second time. If a job needs explicit approval, `queued` is first triggered when job is created and then for the second time after the job's run is approved. This second trigger is not documented.

Any chance this could be updated? Thanks

**Comment by crohr** [comment]
Also, it would be nice if the `waiting` state was sent in place of the first `queued` event. Currently it sends `queued`, `waiting`, then `queued` when it's approved. This makes little sense and requires a lot of custom logic to handle.
